### ISSUE TITLE
[RN 0.74] - Video compressed on Android do not have sound 

### ISSUE THREAD

**Issue by jblarriviere** [numandev1/react-native-compressor#289]
### Current behavior

I'm experiencing the same behavior as #141 and #163 since I've upgraded to RN 0.74
Could the update cause a regression in Android video compression?

### Expected behavior

Video have sound after compression and can be played in chrome

### Platform

- [X] Android

### React Native Version

0.74

### React Native Compressor Version

1.8.24

### Reproducible Steps And Demo

```ts
const compressedPath = await VideoCompressor.compress(path, {}, (compressionProgress) => {
              //whatever
            }),
```

Video file available at `compressedPath` does not have sound. It makes it impossible to play in Chrome browser once uploaded to a server.

**Comment by noelsp** [comment]
Hi, maybe your are experiencing this same issue: https://github.com/numandev1/react-native-compressor/issues/268

**Comment by Titozzz** [comment]
https://github.com/numandev1/react-native-compressor/pull/290

**Comment by numandev1** [comment]
released in `1.8.25`
